presets/airbnb.json

```diff
@@ -3,12 +3,6 @@
     "disallowSpacesInNamedFunctionExpression": {
```

Comment:
I think this rule should be retained; named function expressions seem to be handled differently from anonymous ones in Airbnb's style guide:

```javascript
// good
const x = function () {};
const y = function a() {};
```

Where `x` is considered an example of an anonymous function expression (where the space before open paren is required) and `y` is considered a named function expression (where the space before open paren is disallowed).